### PR TITLE
Update Gulzar documentation

### PR DESCRIPTION
**Pull request by chrissimpkins**
Changes

- Add Fiona Ross to Designers list
- Update repository URL metadata to googlefonts/gulzar

### TODO

- [ ] The https://github.com/google/fonts/blob/main/catalog/designers/bornaizadpanah/bio.html bio has updated information that is current in main but has not been pushed to production, let's add it to the production push
- [ ] https://github.com/google/fonts/blob/main/catalog/designers/bornaizadpanah/bornaizadpanah.png is an updated Designer photo that is current in main but has not been pushed to production, let's add it to the production push.

cc @BornaIz

### CODE REVIEW

File: ofl/gulzar/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -11,5 +11,5 @@
     You can read more about the inspiration, design, and engineering of Gulzar at <a href="https://gulzarfont.org/">gulzarfont.org</a>.
 </p>
 <p>
-    To contribute, see <a href="https://github.com/simoncozens/Gulzar/">github.com/simoncozens/Gulzar/</a>.
+    To contribute, see <a href="https://github.com/googlefonts/Gulzar/">github.com/googlefonts/Gulzar/</a>.
```
Comment (davelab6) [inline ofl/gulzar/DESCRIPTION.en_us.html:14]: Normally I leave the final period on this sentence

**Review by davelab6 — APPROVED** [review]

File: ofl/gulzar/DESCRIPTION.en_us.html (hunk 1 above)
Comment (vv-monsalve) [reply ofl/gulzar/DESCRIPTION.en_us.html:14]: It seems it is there

**Comment by vv-monsalve** [comment]
> * bio has updated information that is current in main but has not been pushed to production, let's add it to the production push

It's included on the [production list ](https://github.com/google/fonts/blob/9168430c0fc5a0a8aa0321ca4ddb93ad061d320c/to_production.txt)that Rod has pushed this week but will actually hit production next week. Hence part of the project list we kept to review/confirm afterward.

**Comment by chrissimpkins** [comment]
@vv-monsalve This has not landed in production yet.  Will you please add these changes to the next production update?  Thanks

**Pull request merged**